.github/workflows/dep_check.yml

```diff
@@ -0,0 +1,19 @@
+# See https://semgrep.dev/explore to learn about what the checks do.
+name: DepCheck
+on: [pull_request]
+jobs:
+  semgrep:
```

Comment:
Minor: Since dependency-check is unrelated to semgrep, we might want to rename this.

Reply:
Done.